scripts/devenv-builder/configure-vm.sh

```diff
@@ -35,16 +41,18 @@ if [ "$(whoami)" != "microshift" ] ; then
 fi
 
 # Check the subscription status and register if necessary
-if ! sudo subscription-manager status >& /dev/null ; then
-   sudo subscription-manager register
+if [ $RHEL_SUBSCRIPTION = true ] ; then
```

Comment:
"true" and "false" are linux commands, so this can just be 
```suggestion
if $RHEL_SUBSCRIPTION ; then
```